assets/javascripts/discourse/templates/components/reviewable-sift-post.hbs

```diff
@@ -0,0 +1,11 @@
+{{reviewable-topic-link reviewable=reviewable tagName=''}}
+<div class='post-contents-wrapper'>
+  {{reviewable-created-by user=reviewable.target_created_by tagName=''}}
+  <div class='post-contents'>
+    {{reviewable-created-by-name user=reviewable.target_created_by tagName=''}}
+    <div class='post-body'>
+      {{{ reviewable.payload.post_cooked }}}
+    </div>
```

Comment:
This seems to be missing the custom sift rendering. For example, the report by word. I believe it's in `sift-flag-queue.hbs`:

![image](https://user-images.githubusercontent.com/17538/55560384-1e155400-56be-11e9-94ca-412db1edc19e.png)